### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by StaceyLouis**
Added Landing Page Component.
added NavBar Component
will update styles as Ant Design needs to be implemented
added Ant Design  dependency
added styled components dependency

### CODE REVIEW

**Review by code-dependent — APPROVED** [review]
Its looking good.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-      - uses: paambaati/codeclimate-action@v2.6.0
-        env:
-          CC_TEST_REPORTER_ID: ${{ secrets.CodeClimateReporterId }}
-        with:
-          # Run our `coverage` script in our `package.json`
-          coverageCommand: npm run coverage
-          # By default, this looks for a `coverage` folder in the root of your project, but you may need to change this
-          coverageLocations: './coverage/clover.xml:clover'
-          # Great for troubleshooting
-          debug: true
```
Comment (buchananta) [inline .github/workflows/ci.yml:27]: This file is setup so github does automated testing for us doesn't it? Why remove it?

File: package.json [hunk 2]
```diff
@@ -64,27 +64,5 @@
...
-    "eslint-plugin-react-hooks": "^4.0.4",
-    "husky": "^4.2.5",
-    "jest-canvas-mock": "^2.2.0",
-    "jest-environment-jsdom-sixteen": "^1.0.3",
-    "jest-prop-type-error": "^1.1.0",
-    "kleur": "^4.1.1",
-    "lint-staged": "^10.2.11",
-    "prettier-eslint-cli": "^5.0.0",
-    "prompts": "^2.3.2",
-    "yaml": "^1.10.0"
```
Comment (buchananta) [inline package.json:88]: We need all of these devDependencies for testing don't we?
Specifically our engineering guidelines talk about the [react-testing-library ](https://docs.labs.lambdaschool.com/labs-spa-starter/testing) and [storybook](https://docs.labs.lambdaschool.com/labs-spa-starter/storybook). I remember eslint being mentioned as well.

File: src/components/Landing-page/profile-card.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
...dv22uIEiLA4q7nvvCug8WTqzQveOH26fodo7g6uFe/a17W3+nFBAkRYENRdb1vkkz1CH9cPsVy/jrhr27PqMYvENYNlHAIesRiBYwRy0V+8iXP8+/fvX11Mr7L7ECueb/r48eMqm7FuI2BGWDEG8cm+7G3NEOfmdcTQw4h9/55lhm7DekRYKQPZF2ArbXTAyu4kDYB2YxUzwg0gi/41ztHnfQG26HbGel/crVrm7tNY+/1btkOEAZ2M05r4FB7r9GbAIdxaZYrHdOsgJ/wCEQY0J74TmOKnbxxT9n3FgGGWWsVdowHtjt9Nnvf7yQM2aZU/TIAIAxrw6dOnAWtZZcoEnBpNuTuObWMEiLAx1HY0ZQJEmHJ3HNvGCBBhY6jtaMoEiJB0Z29vL6ls58vxPcO8/zfrdo5qvKO+d3Fx8Wu8zf1dW4p/cPzLly/dtv9Ts/EbcvGAHhHyfBIhZ6NSiIBTo0LNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiEC/wGgKKC4YMA4TAAAAABJRU5ErkJggg=="
```
Comment (buchananta) [inline src/components/Landing-page/profile-card.js:16]: Do we need to embed an image directly into the html?

**Review by buchananta — CHANGES_REQUESTED** [review]
Looks good! Only issue I have is deleting the testing, I think we're required to keep that, so even if we removed it now, we'd just have to add it back later.

**Pull request closed without merging**